### PR TITLE
feat: add velero pvc backup scheduler addon

### PR DESCRIPTION
**Pull request by PedroRossi**

### CODE REVIEW

File: charts/velero-pvc-backup-scheduler/templates/schedule.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: velero.io/v1
+kind: Schedule
+metadata:
+  name: {{ template "application.name" . }}
+  namespace: {{ .Values.veleroNamesace }}
```
Comment (jfmrm) [inline charts/velero-pvc-backup-scheduler/templates/schedule.yaml:5]: can't we have the schedule at the same namepsace as the application? If so we can omit the namespace setting

**Review by jfmrm — APPROVED** [review]

File: charts/velero-pvc-backup-scheduler/values.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Velero install namespace
+veleroNamespace: addons
+# The cron for running the backups
+backupScheduleCron: 0 0,8,16 * * *
+# The amount of time before backups created on this schedule are eligible for garbage collection.
+backupGarbageCollectionTTL: 7days
```
Comment (jfmrm) [inline charts/velero-pvc-backup-scheduler/values.yaml:6]: does this mean that the backups will be retained for 7 days, then they will be deleted?

File: charts/velero-pvc-backup-scheduler/templates/schedule.yaml (hunk 1 above)
Comment (PedroRossi) [reply charts/velero-pvc-backup-scheduler/templates/schedule.yaml:5]: No, it needs to run at the same namespace as the velero application is installed, we can force this on the base overlay if needed

File: charts/velero-pvc-backup-scheduler/values.yaml (hunk 2 above)
Comment (PedroRossi) [reply charts/velero-pvc-backup-scheduler/values.yaml:6]: Yes

**Pull request merged**